Detectors/TOF/calibration/src/TOFFEElightReader.cxx

```diff
@@ -22,30 +22,53 @@ void TOFFEElightReader::loadFEElightConfig(const char* fileName)
   char* expandedFileName = gSystem->ExpandPathName(fileName);
   std::ifstream is;
   is.open(expandedFileName, std::ios::binary);
-  is.read(reinterpret_cast<char*>(&mFEElightConfig), sizeof(mFEElightConfig));
+  char* tempbuf = new char[sizeof(o2::tof::TOFFEElightConfig)];
+  is.read(tempbuf, sizeof(o2::tof::TOFFEElightConfig));
   is.close();
+  mFEElightConfig = reinterpret_cast<TOFFEElightConfig*>(tempbuf);
+}
+
+//_______________________________________________________________
+void TOFFEElightReader::loadFEElightConfig(gsl::span<const char> configBuf)
+{
+  // load FEElight config from buffer
+
+  if (configBuf.size() != sizeof(o2::tof::TOFFEElightConfig)) {
+    LOG(FATAL) << "Incoming message with TOFFEE configuration does not match expected size: " << configBuf.size() << " received, " << sizeof(*mFEElightConfig) << " expected";
+  }
+  mFEElightConfig = const_cast<TOFFEElightConfig*>(reinterpret_cast<const TOFFEElightConfig*>(configBuf.data()));
```

Comment:
you don't need const_cast, instead, better define `const TOFFEElightConfig* mFEElightConfig = nullptr`

Reply:
I also thought about this possibility, but it was simpler to add the `const_cast` :-) 
Changed.